Gemfile

```diff
@@ -10,7 +10,7 @@ gem 'bootstrap-sass'
 gem 'carrierwave'
 gem 'cocoon'
 gem 'coffee-rails', '~> 4.0.0'
-gem 'crowdint_auth', '0.1.3'
+gem 'crowdint_auth', '0.1.3', github: 'jmoralesc/crowdint_auth_new'
```

Comment:
Agree with @softr8... @jmoralesc do you know if your PR were merged into crowdint_auth_new from original repo?